src/renderer/components/ft-prompt/ft-prompt.vue

```diff
@@ -28,9 +28,19 @@
         <ft-flex-box>
           <ft-button
             v-for="(option, index) in optionNames"
+            :id="'prompt-' + removeWhitespace(label) + '-' + index"
             :key="index"
             :label="option"
             @click="$emit('click', optionValues[index])"
+            @keydown.left.prevent="focusItem(index-1)"
+            @keydown.right.prevent="focusItem(index+1)"
```

Comment:
This probably needs handling (wasn't able to test my theory due to the error above) for when the left key is pressed on the first button and the right key is pressed on the last button in the prompt.